data/items/item.js

```diff
@@ -0,0 +1,785 @@
+    '971', '991', '998',
+    '992', '971', '959',
+    '957', '929', '923',
+    '939', '979', '932',
+    '973', '938',
+  ],
+  [
+    '963', '909', '913',
+    '984', '953', '929',
+    '925', '948', '924',
+    '970', '946', '924',
+    '915',
+  ],
+  [
+    '980', '945', '942',
+    '941', '998', '914',
+    '978', '945', '956',
+    '971', '979',
+  ],
+  [
+    '958', '934', '916',
+    '922', '955', '941',
+    '941', '960', '900',
+    '907',
+  ],
+  [
+    '916', '923',
+    '940', '926',
+    '923', '938',
+    '997', '914',
+    '981',
+  ],
+  [
+    '942', '909', '950',
+    '902', '944', '972',
+    '960', '913', '968',
+    '995', '981', '962',
+    '923', '940', '976',
+  ],
+  [
+    '966', '966', '927',
+    '968', '973', '899',
+    '923', '913', '971',
+    '990', '992', '929',
+    '947', '921', '968',
+    '933',
+  ],
+  [
+    '913', '982', '948',
+    '906', '917', '965',
+    '977', '965', '905',
+    '932',
+  ],
+  [
+    '995', '935', '946',
+    '926', '972', '899',
+    '988', '946', '957',
+    '979', '969', '969',
+    '988', '924', '943',
+    '991',
+  ],
+  [
+    '936', '952', '901',
+    '924', '941', '960',
+    '979', '905', '922',
+    '936', '989', '903',
+    '912', '931', '929',
+    '911', '948',
+  ],
+  [
+    '990', '954', '977',
+    '931', '997', '941',
+    '928', '966', '991',
+    '930', '906', '960',
+    '918', '965', '997',
+    '982', '967', '949',
+  ],
+  [
+    '941', '908',
+    '900', '991',
+    '945', '972',
+    '932', '956',
+    '946',
+  ],
+  ['925', '975', '964', '956', '956'],
+  [
+    '910', '961', '984',
+    '924', '960', '915',
+    '996', '919', '998',
+    '932', '984', '912',
+    '945', '964', '927',
+    '959',
+  ],
+  [
+    '917', '907',
+    '921', '994',
+    '920', '986',
+    '958',
+  ],
+  [
+    '948', '988', '951',
+    '902', '914', '951',
+    '918', '913', '920',
+    '924', '923', '938',
+    '926', '990', '984',
+    '948',
+  ],
+  [
+    '962', '960', '923',
+    '931', '971', '968',
+    '925', '933', '922',
+    '924', '902', '959',
+    '990', '919', '922',
+    '952', '996', '935',
+  ],
+  [
+    '932', '926', '906',
+    '971', '979', '924',
+    '900', '915', '995',
+    '916', '945', '931',
+  ],
+  [
+    '940', '992', '975',
+    '976', '987', '936',
+    '907', '997', '985',
+    '981', '911', '947',
+    '962', '989', '968',
+  ],
+];
+
+module.exports = idNumber.map((id, i) => ({
+  _id: id,
+  name: productName[i],
+  imageUrl: 'http://placeimg.com/640/480/fashion',
+  shopName: faker.helpers.randomize(shopNames),
+  price: {
+    display: `$${priceValues[i]}`,
+    worth: priceValues[i],
+    onSale: faker.helpers.randomize([true, false]),
+    salePercentage: faker.helpers.randomize([0.10, 0.15, 0.20, 0.25, 0.30, 0.50, 0.75]),
+    salePrice: 0,
+  },
+  shipping: {
+    display: faker.helpers.randomize(shipping),
+    eligibility: true,
+  },
+  recommended: {
+    fromShop: [],
+    similarProduct: recommend[i],
+  },
+}));
```

Comment:
did you manually input all of the recommended arrays? We need to discuss product ID's